robottelo/hosts.py

```diff
@@ -2057,6 +2058,10 @@ def register_host_custom_repo(self, module_org, rhel_contenthost, repo_urls):
             # refresh repository metadata on the host
             rhel_contenthost.execute('subscription-manager repos --list')
 
+        # Override the repos to enabled  if needed
+        if override:
+            rhel_contenthost.execute(r'subscription-manager repos --enable \*')
+
```

Comment:
Custom repos are disabled by default, regardless AK existence. If you register a host using this function, they are present/visible on both sides, but disabled. This enables them on both sides too, so it seems valid way to me. Unless you have some nicer way.

Reply:
what do you mean by both sides here? host and ? 
Currently, `register_host_custom_repo` uses the old registration method, instead we should be using new Global registration method which will work with AK and we could easily override the repos for host in AK, but it feels out of scope of this PR :)

Reply:
Both sides I mean - Host itself (in `/etc/yum.repos.d`) and Satellite (in host details).
For host registration rewrite - I think it would deserve a separate PR.

Reply:
yes, definately, can you create an issue for this? so we don't miss this :)